core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -361,8 +374,9 @@ class CityInfo {
         cityConstructions.endTurn(stats)
         expansion.nextTurn(stats.culture)
         if (isBeingRazed) {
-            population.addPopulation(-1)
-            if (population.population <= 0) { // there are strange cases where we get to -1
+            val removedPopulation = 1 * civInfo.getMatchingUniques("Cities are razed [] times as fast").sumBy { it.params[0].toInt() }
```

Comment:
Um, wouldn't this mean that if you have no such uniques, removedPopulation would be 0...?

Reply:
Good catch. And do we really want a "2 times" and a "3 times" to stack to "5 times"? Also, a textbook case for `.fold(1)`?